docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+```
+
+You'll notice we've kept a few things that will be immediately useful: 
+*  `<AvSphereHandle />`, because we'll want to interact with our gadget - no sense in losing the handler. 
+*  `<AvTransform>` to easily manipulate our panel, as well as the contained panel.
+
+Aside from that, and new empty WatchState object and a few naming tweaks, this is a barebones version of the default gadget. Let's get to work!
+
+First, right before the final `</div>` tag in the `render` method, let's add a simple sanity check to make sure our dev environment is fully functional and the gadget is being updated:
+
+```tsx
+<div className="Watch">
+	Hello Aardvark!
+</div>
+```
+Upon saving and instantiating a new gadget, you should be able to see immediate results, but they'll default to the size of fairly small chicken scratch. In `styles.css`, let's fix that!
+
+```css
+.Watch
+{
+	font-size: 4rem;
+}
+```
+![hello-aardvark](./media/hello-aardvark.jpg)
+
+Now we're cooking with fire. Let's get it looking like a proper digital watch. We'll tweak the CSS of the `.Watch` class so that the numbers utilize a proper digital font (I picked the freely available [Digital 7](https://www.1001fonts.com/digital-7-font.html), and position it somewhere that'll make sense once we add the digital watch model.
+
+A nice trick here is making the panel background transparent, because we don't want a "page" as much as "digits displayed on a watch's screen". Here's `styles.css`:
+
+```css
+@font-face {
+	font-family: 'digital-7';
+	src: url('./fonts/digital-7\ \(mono\).ttf')
+}
+
+body, html {
+	background-color: transparent;
+}
+
+.Watch {
+	font-family: 'digital-7', monospace;
+	font-size: 2.8rem;
+	color: #4Af626;
+	margin-top: 47%;
+	margin-left:48%;
+	max-width: 1.5rem;
+	text-align: center;
+}
+```
+et voilà!
+
+![hello-aardvark-digital](./media/hello-aardvark-dig.jpg)
+
+
+# Step 3: _Time After Time_ - Adding the Watch model
+
+After scouring Poly and Sketchfab for good, free digital watch models, I found this [great-looking, downloadable photogrammetized model of a Casio G-Shock](https://sketchfab.com/3d-models/casio-g-shock-digital-watch-b6466b64dd8b495c99f24a5054491f4d) by [jeandiz](https://sketchfab.com/jeandiz). Clocking in at ~25MB, its textures are way too nice and detailed for our purposes, so I've resized them in imagemagick and did some model cleanup in Blender; you can find the optimized, <1MB gltf file [here](media/watch.glb). 
+
+![Casio](media/casio.jpg)
+
+Let's add this make this the gadget preview model while we're at it! in `manifest.webmanifest`, under `icons`, change to:
+```webmanifest
+	"icons": [
+		{
+			"src": "models/watch.glb",
+			"type": "model/gltf-binary"
+		}
+	],
+```
+**important** - remember to add the watch (and all gltf models) to your **`dist/models`** folder, not `src/models`! gltf models do not get copied over as part of the compilation flow - the path is relative from `dist/manifest.webmanifest`.
```

Comment:
I think I understand out other conversation better now. Models can certainly be copied over automatically. You just need to to edit webpackconfig.js

Reply:
absolutely, big a-ha moment re. workflow. Thanks again.